src/vector/indexedbd-worker.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2017 Vector Creations Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The prescribed way of doing this import would be:
+//import {IndexedDbStoreWorker} from 'matrix-js-sdk';
+// However, this still pulls in all of the js-sdk and we only use a tiny fraction
+// of it. It also causes an Olm error to appear because we don't have an Olm in scope.
+// Instead, we do this:
+import IndexedDbStoreWorker from 'matrix-js-sdk/lib/store/indexeddb-remote-worker';
```

Comment:
Please don't advocate this style. I understand why you're doing it this case, but it prevents us from changing the position of files in the JS SDK. I don't believe we do pull in all of the JS SDK because of the optimisation pass which babel goes through in production mode. We should fix the Olm error rather than work around it like this.

It sounds like it will work if you do it properly, so please just do that.